src/ui/screens/home/controller/ControllerView.qml

```diff
@@ -635,7 +634,6 @@ Item {
             objectName: "controllerSubTitle"
 
             lineHeight: MZTheme.theme.controllerInterLineHeight
-            Accessible.ignored: true
```

Comment:
Q: Should this have the same `Accessible.ignored` rule as `logoTitle` or do we maybe get this for free when we change visibility to false during `StateOn`?

Reply:
Yes, `logoSubtitle`'s visibility is turned off during `StateOn`, and `MZHeadline` already has `Accessible.ignored: !visible`. 
`logoTitle` needs the additional conditions in `Accessible.ignored` because it is visible during `StateOn` and when the the Connection Info screen or IP Info Panel is open.

Reply:
cool, thanks for the explainer!